apps/site/assets/ts/stop/components/DepartureList.tsx

```diff
@@ -28,10 +37,28 @@ const DepartureList = ({
   );
 
   let departures: DepartureInfo[] = [];
+  const groupedAlerts = alertsByRoute(alerts);
+  const alertsForRoute = groupedAlerts[route.id] || [];
 
+  const alertsByDirectionObj = alertsByDirectionId(alertsForRoute);
+  const alertsAffectingBothDirectionsArray = alertsAffectingBothDirections(
+    alertsForRoute
+  );
+
+  const alertsDirectionArray = alertsByDirectionObj[directionId]
+    ? alertsByDirectionObj[directionId]
+    : [];
+
+  const allAlerts = concat(
+    alertsAffectingBothDirectionsArray,
+    alertsDirectionArray
+  ).filter(alert => {
+    return ["detour", "suspension", "shuttle"].includes(alert.effect);
+  });
   // TODO: handle no predictions or schedules case and predictions only case
   return (
     <>
+      <Alerts alerts={allAlerts} />
       {schedules.length && (
```

Comment:
question:  Do we want to display schedules if the route is suspended?  Will we have schedules to display then?

Reply:
If the route is suspended it's probably best not show schedules. Will add a check. They show on `dev-green` now and it's a bit confusing to see a closed stop with schedules https://dev-green.mbtace.com/stops/15566